host/src/PrelaunchApp/App.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net7.0;net8.0</TargetFrameworks>
```

Comment:
why not net6.0 once the Host moves to net8.0?

Reply:
Added net6.0